mkdocs.yml

```diff
@@ -21,11 +21,16 @@ nav:
       - Glossary: 'overview/glossary.md'
       - Logo assets: 'overview/logos.md'
   - Getting Started:
-      - Getting Started: 'getting-started/index.md'
-      - Create an App: 'getting-started/create-an-app.md'
-      - App configuration:
+      - Installing a standalone server (Developer or Admin): 'getting-started/standalone-install.md'
```

Comment:
Can we keep name this something like `Creating an App` or `Introduction`? 

and keep it at the `index.md` route so that we don't break links?